bin/epaa.py

```diff
@@ -729,15 +729,22 @@ def create_affinity_values(allele, length, j, method, max_scores, allele_strings
         return np.nan
 
 
-def create_binder_values(aff, method):
+def create_binder_values(aff, method, thresholds):
     if not pd.isnull(aff):
-        if 'syf' in method:
-            return True if aff > 50.0 else False
-        else:
-            return True if aff <= 500.0 else False
+        return True if aff > thresholds.get(method) else False
     else:
         return np.nan
 
+
+def assign_tool_thresholds(methods, tool_thresholds):
```

Comment:
I was just thinking if it might be better if we expect `json` as parameter value, like:

```
{"methodx": threshold1, "methody": threshold2}
```

This wouldn't be dependent on the order of provided thresholds. What do you think?

Reply:
Thats more precise, I like it! We then don't need this function anymore

Reply:
We should still check somewhere if thresholds are provided for all methods that are specified. Otherwise it will fail.